nam/data.py

```diff
@@ -122,14 +122,14 @@ def np_to_wav(
     filename: Union[str, Path],
     rate: int = 48_000,
     sampwidth: int = 3,
-    scale="none",
+    **kwargs
```

Comment:
Can I get this to be backwards-compatible based on the detected version of wavio? I'm fairly sure that this will cause bug reports w/ the users if it's not super-hand-holdy.

`wavio.__version__` exists, right?  If so, please check `kwargs` and provide defaults accordingly.